content/blog/how-elevate-supports-business-needs.md

```diff
@@ -0,0 +1,182 @@
+
+So I had to move the /boot partition to a new partition which was bigger, again, something to keep in mind,
+while ELevate is a great tool to help deprecating those last CentOS 6 servers, it still might be cleaner to do it from scratch some day.
+
+While moving the partition is quite an easy way, it can be quite dirty if you put it directly behind your data partition,
+you might want to consider moving that further back and just expanding the current /boot partition itself,
+or even simpler, move it to a seperate disk, for which the steps are as follows:
+
+```bash
+# add a new disk of 1Gb to your VM
+# add it to the boot order in your hypervisor of choice
+
+# create a new partition on the new disk with fdisk /dev/sdb as primary and type Linux
+# be sure to use a starting point for the partition a bit away from the first sector, otherwise there is not enough space left for grub2 once you get to AlmaLinux 8, for example cylinder 64 or sector 2049
+
+# reload the partitions and create a ext4 filesystem on the new partition
+partx -v -a /dev/sdb
+mkfs.ext4 /dev/sdb1
+
+# copy your original /boot partition to the new partition with dd
+dd if=/dev/sda1 of=/dev/sdb1 bs=1M
+
+# extend the new partition as the old size is inherited while using dd
+e2fsck -fy /dev/sdb1
+resize2fs /dev/sdb1
+e2fsck -fy /dev/sdb1
+
+# unmount the old boot partition and mount the new one
+umount /dev/sda1
+mount /dev/sdb1 /boot
+
+# install grub on the new disk and scramble uuid of original /boot so it doesn't get mounted accidentally
+grub-install --recheck /dev/sdb
+tune2fs /dev/sda1 -U random
+
+# check the UUID's and grub
+blkid
+cat /etc/fstab
+cat /boot/grub/grub.conf
+
+# delete old partition so it can't be used again accidentally
+fdisk /dev/sda
+
+# reboot the machine to actually boot from the new /boot partition
+reboot now
+```
+
+### grub to grub2!
+When upgrading to CentOS 7 my test server switches from grub to grub2, when I wanted to try ELevating it to AlmaLinux 8 it complained about some grub issues,
+I had to create a /boot/grub2/grub.cfg file and create /etc/default/grub.
+
+For /etc/default/grub I created it by using the booted system and put the following there:
+```bash
+GRUB_TIMEOUT=5
+GRUB_DISTRIBUTOR="$(sed 's, release .*$,,g' /etc/system-release)"
+GRUB_DEFAULT=saved
+GRUB_DISABLE_SUBMENU=true
+#GRUB_TERMINAL_OUTPUT="console"
+GRUB_CMDLINE_LINUX="< put the output of cat /proc/cmdline here! >"
+GRUB_DISABLE_RECOVERY="true"
+GRUB_THEME="/boot/grub2/themes/system/theme.txt"
+```
+
+And generate /boot/grub2/grub.cfg with:
+```bash
+grub2-mkconfig -o /boot/grub2/grub.cfg
+```
+
+### DirectAdmin specific tips
+After every leapp upgrade we rebuild all packages that DirectAdmin brings, sometimes this can go wrong with for example some older local libraries, the following commands might help you out sometime.
```

Comment:
```suggestion
After every `leapp upgrade,` we rebuild all packages that DirectAdmin includes. Sometimes this can go wrong, for example with some older local libraries. The following commands might help you out, too.
```